languages/go/example/example.go

```diff
@@ -112,5 +112,26 @@ func main() {
 
 	fmt.Println(string(jsonSecrets))
 
+	// Generate a password which can be used as a secret value
+	request := sdk.PasswordGeneratorRequest{
+		AvoidAmbiguous: true,
+		Length:         64,
+		Lowercase:      true,
+		MinLowercase:   new(int64),
```

Comment:
new(int64) is 0 ?

Reply:
Yes, just putting it in there as an example